### PR TITLE
Favor async/await pattern over .Result for acceptance tests

### PR DESCRIPTION
**Pull request by Evangelink**
## Description

Ensure acceptance tests favor async/await pattern over .Result

### CODE REVIEW

File: test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs [hunk 1]
```diff
@@ -63,16 +63,16 @@ public void CommandLineRunSettingsShouldWinAmongAllOptions(RunnerInfo runnerInfo
                         string.Concat("RunConfiguration.TestAdaptersPaths=" , this.GetTestAdapterPath())
                     });
 
-            this.RunTestWithRunSettings(runConfigurationDictionary, runSettingsArgs, additionalArgs, testhostProcessName, expectedNumOfProcessCreated);
+            await this.RunTestWithRunSettings(runConfigurationDictionary, runSettingsArgs, additionalArgs, testhostProcessName, expectedNumOfProcessCreated);
         }
 
         /// <summary>
-        /// Command line run settings should have high precedence btween cli runsettings and cli switches.
+        /// Command line run settings should have high precedence between cli runsettings and cli switches.
```
Comment (Evangelink) [inline test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs:70]: Let me know if you prefer this typo fix to be in a separate PR.

Comment (nohwnd) [reply test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs:70]: It's fine to include somewhere. You don't have to start a new PR for each change :)

**Comment by nohwnd** [comment]
Closed because the async task needs to run on background to do work, and only be awaited after cancellation to collect the results.
